frontend/public/forbidden-page.html

```diff
@@ -0,0 +1,156 @@
+                    c0.2-0.3,0.2-0.6,0-0.9h-0.8c-0.2,0-0.5-0.1-0.6-0.3c-0.2-0.4-0.2-0.9-0.1-1.4v-6.3l3.4,8.4l0,0c0.1,0.2,0.3,0.3,0.5,0.3h0.4
+                    l3.9-8.5v6.8c0,0.2,0,0.3,0,0.5l0,0c0,0.2-0.1,0.3-0.2,0.4l-0.6,0.2h-0.8c-0.2,0.3-0.2,0.6,0,0.9h5C127,37,127,36.6,126.7,36.3z
+                     M111.7,26.1h-4.2c-0.2,0.2-0.3,0.5-0.1,0.8h1.3c0.2,0.3,0.2,3.5,0.2,5.2s-1.6,4-3.1,4s-3-1.2-3-4v-3.7c0-1.3,0-1.4,0.7-1.4h0.6
+                    c0.2-0.1,0.3-0.2,0.2-0.5s-0.1-0.4-0.3-0.4h-4.6c-0.1,0.1-0.2,0.3-0.2,0.5c0,0.2,0.1,0.3,0.2,0.4h0.7c0.7,0,0.7,0,0.7,1.6v3.9
+                    c-0.1,1.5,0.5,3,1.6,4c0.9,0.6,1.9,1,3,1h0.3c1.3,0,2.5-0.5,3.5-1.3c1-1.1,1.1-2.1,1.1-4.1c0-1.6,0.1-3.3,0.2-4.9
+                    c0.1-0.1,0.3-0.2,0.5-0.2h0.7c0.2-0.1,0.2-0.2,0.2-0.5C111.9,26.3,111.8,26.2,111.7,26.1z M76.9,10.4c0.6,0,0.7,0.1,0.7,1.5v7.2
+                    c0,1.9,0,1.9-0.9,2.1h-0.6c-0.2,0.2-0.3,0.5-0.1,0.8h5.2c3.7,0,5.8-1.5,5.8-4.1c-0.1-1.6-1.3-3-2.9-3.2c1.1-0.4,1.8-1.4,1.8-2.6
+                    c-0.1-1-0.6-1.8-1.5-2.3c-1-0.4-2-0.6-3.1-0.6h-0.5l-4.5,0.2c-0.1,0-0.3,0.2-0.3,0.3c-0.1,0.3,0,0.5,0.2,0.7H76.9z M84.7,18.7
+                    c0,1.9-1.6,2.3-3,2.3h-0.2c-0.4,0-0.9-0.1-1.2-0.4c-0.3-0.5-0.4-1.2-0.4-1.8V16h0.9C84.2,15.8,84.7,17.7,84.7,18.7z M79.9,10.5h1
+                    c1.3,0,2.8,0.6,2.8,2.2s-0.5,2-3,2.1h-0.8v-4.1C79.9,10.6,79.9,10.6,79.9,10.5z M77.6,33.6h-0.5c-0.5,1.6-2,2.6-3.6,2.6h-0.2
+                    c-3.3,0-5.5-2.5-5.5-6.1c-0.2-2.9,2.1-5.4,5-5.6c0.1,0,0.1,0,0.2,0c2.5,0,3.3,0.9,3.7,2.6h1.2c-0.2-0.9-0.3-1.9-0.3-2.8V24h-0.7
+                    c-1.1-0.3-2.3-0.5-3.5-0.5H73c-1.9,0-3.7,0.6-5.2,1.7c-1.6,1.3-2.6,3.4-2.5,5.5c-0.1,2.2,1,4.4,2.9,5.6c1.4,0.9,3,1.4,4.7,1.4h0.3
+                    c1.1,0,2.2-0.2,3.2-0.5L77,37h0.2c0.5-0.9,0.9-1.8,1-2.8l0,0C78.2,33.9,77.9,33.6,77.6,33.6z M135.3,31c1.2-0.5,1.8-1.9,1.3-3.1
+                    c-0.3-0.7-0.8-1.2-1.4-1.4c-1-0.3-2-0.4-3-0.4h-4.3l-0.3,0.2c-0.1,0.2-0.1,0.5,0.1,0.7h0.8c0.6,0,0.6,0.1,0.6,1.3v6.3
+                    c0,1.6,0,1.6-0.7,1.7h-0.5c-0.1,0.1-0.2,0.2-0.2,0.4c0,0.2,0.1,0.4,0.2,0.5h4.7c4.2,0,5.2-0.9,5.2-3.6c-0.2-1.3-1.3-2.4-2.6-2.6
+                    L135.3,31z M131.2,27h0.9c1.2,0,2.6,0.5,2.6,1.9s-0.6,1.8-2.8,1.8h-0.7C131.2,30.7,131.2,27,131.2,27z M131.6,36
+                    c-0.3-0.2-0.5-1-0.5-1.5v-2.7h1c2.8,0,3.4,1.4,3.4,2.6s-1.5,1.9-2.8,1.9C132.3,36.3,131.9,36.2,131.6,36L131.6,36z"/>
+                            <rect x="116.9" y="36.6" class="st3" width="0.6" height="0.6"/>
+            </svg>
+            <div class="v-toolbar__title bcgov-title">
+             Water Allocation
+            </div>
+        </div>
+    </header>
+</div>
+<div>
+    <div class="message">
+        <p>
+            Welcome to the Water Allocation (WALLY) application.
+            It looks like you need to be granted access.
+        </p>
+        <p>
+            Please <a href="mailto:Jake.Morris@gov.bc.ca?subject=Access Request to WALLY">contact us</a> for help.
```

Comment:
Does Wally have a shared email inbox like GWELLS? If not, then using the service owner's email might be a better idea here.